packages/server/lib/mud/setupNetwork.ts

```diff
@@ -4,11 +4,13 @@ import { defineContractComponents } from "./contractComponents";
 import { world } from "./world";
 import { Contract, Signer, utils } from "ethers";
 import { JsonRpcProvider } from "@ethersproject/providers";
-import { IWorld__factory } from "contracts/types/ethers-contracts/factories/IWorld__factory";
+import * as worldFactory from "contracts/types/ethers-contracts/factories/IWorld__factory";
 import { getTableIds } from "@latticexyz/utils";
 import storeConfig from "contracts/mud.config";
 import { setupMUDV2Network } from "@latticexyz/std-client";
 
+const { IWorld__factory } = (worldFactory as any).default.IWorld__factory;
```

Comment:
unclear what's nodejs' problem here is, but using the named import directly fails while this seems to work